### PR TITLE
[Feature][Connector-V2] Ftp file sink suport multiple table and save mode

### PR DESCRIPTION
**Pull request by chl-wxp**
### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-ftp-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/ftp/FtpFileIT.java [hunk 1]
```diff
@@ -143,6 +153,70 @@ public void testFtpFileReadAndWrite(TestContainer container)
         helper.execute("/excel/fake_source_to_ftp_root_path_excel.conf");
     }
 
+    @TestTemplate
+    @DisabledOnContainer(
+            value = {},
+            type = {EngineType.FLINK},
+            disabledReason =
+                    "Fink test is multi-node, FtpFile connector will use different containers for obtaining files")
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-ftp-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/ftp/FtpFileIT.java]:
```suggestion
                    "Flink dosen't support multi-table at now")
```

File: docs/en/connector-v2/sink/FtpFile.md [hunk 2]
```diff
@@ -83,7 +85,7 @@ The target ftp password is required
 
 ### path [string]
 
-The target dir path is required.
+The target dir path is required, you can inject the upstream CatalogTable into the path by using: `${database_name}`, `${table_name}` and `${schema_name}`.
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/FtpFile.md]:
```suggestion
The target dir path is required.
```
We have `Sink Parameter Placeholders` part to share this feature.

File: docs/en/connector-v2/sink/FtpFile.md [hunk 3]
```diff
@@ -273,6 +287,33 @@ FtpFile {
 
 ```
 
+When our source end is multiple tables, and wants different expressions to different directory, we can configure this way
+```bash
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/FtpFile.md]:
```suggestion
```hocon
```

Comment (chl-wxp) [reply docs/en/connector-v2/sink/FtpFile.md]: updated

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by hailin0 — APPROVED** [review]

**Pull request merged**